monai/apps/deepgrow/transforms.py

```diff
@@ -702,10 +722,11 @@ def __call__(self, data):
         # update bounding box in case it was corrected by the SpatialCrop constructor
         box_start = np.array([s.start for s in cropper.slices])
         box_end = np.array([s.stop for s in cropper.slices])
-        for key in self.key_iterator(d):
+        for key, meta_key, meta_key_postfix in self.key_iterator(d, self.meta_keys, self.meta_key_postfix):
             if not np.array_equal(d[key].shape[1:], original_spatial_shape):
                 raise RuntimeError("All the image specified in keys should have same spatial shape")
-            meta_key = f"{key}_{self.meta_key_postfix}"
+            if meta_key is None:
```

Comment:
Elsewhere you have `meta_key = meta_key or f"{key}_{meta_key_postfix}"` which I think is correct to use. The `or` operator would be a problem if the left-hand side was something that overrode certain operators but using it to only check for `None` should be fine.

Reply:
Sure, updated in the latest commit.

Thanks.